assets/airports/airportLoadList.js

```diff
@@ -227,6 +227,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             name: 'Phoenix Sky Harbor International Airport',
             wip: true
         },
+        {
+            icao: 'kpvd',
+            level: 'beginner',
+            name: 'Theodore Francis State Memorial Airport',
+            wip: true
```

Comment:
This will be in full compliance with the specs before merge, so we can just set this to false here.